channels/auth.py

```diff
@@ -28,7 +137,7 @@ def __call__(self, scope):
             raise ValueError("AuthMiddleware cannot find session in scope. SessionMiddleware must be above it.")
         # Add it to the scope if it's not there already
         if "user" not in scope:
-            scope["user"] = SimpleLazyObject(lambda: get_user(scope))
+            scope["user"] = async_to_sync(get_user)(scope)
```

Comment:
You can't do this here, as you'd be doing a blocking operation in the ASGI app's first callable (which can't block). I'm surprised this didn't raise an error during testing... still, you should keep the lambda for now (that means you can still accidentally call it during an async consumer, but I'd like to fix that separately)